addon/templates/components/md-select.hbs

```diff
@@ -1,12 +1,18 @@
 <label id="{{id}}" class="active">{{label}}</label>
-{{view "select"
-  id=id
-  content=content
-  optionValuePath=optionValuePath
-  optionLabelPath=optionLabelPath
-  prompt=prompt
-  value=value
-  classNameBindings="validate:validate errors:invalid:valid"}}
+<select onchange={{action (mut value) value="target.value"}}
+        class="{{if validate 'validate'}} {{if errors 'invalid' 'valid'}}"
+        disabled={{if disabled "true"}}>
+  {{#if prompt}}
+    <option value="" disabled selected>{{prompt}}</option>
+  {{/if}}
+  {{#each content as |option|}}
+    {{#with (if (eq optionValuePath 'content') option (or (get option (md-regex-replace (or optionValuePath 'content.value') '(content\.|^content$)' '')) option)) as |optionValue|}}
+      {{#with (if (eq optionLabelPath 'content') option (or (get option (md-regex-replace (or optionLabelPath 'content.label') '(content\.|^content$)' '')) option)) as |optionLabel|}}
+        <option value={{optionValue}} selected={{if (eq value optionValue) "true"}}>{{optionLabel}}</option>
+      {{/with}}
+    {{/with}}
+  {{/each}}
+</select>
```

Comment:
This is a bit hairy only because I used the `get` helper to extract `optionValue` & `optionLabel`, but wanted to keep backwards compatibility for those who define `optionsValuePath` and `optionLabelPath` with a form of `"content.{{attrName}}" instead of just "{{some-attrName}}".